src/utils/url.js

```diff
@@ -154,3 +154,131 @@ export function stringifyQuery(query) {
 
     return search
 }
+
+/* Import URL utils start */
+
+const ADMIN_URL_REGEXP = new RegExp(
+    '^(ftp|http|https)://(.*(.bgdi.ch|.geo.admin.ch|.swisstopo.cloud)|localhost:[0-9]{1,5})'
+)
+
+// todo use correct URLs
+const PROXY_URL = 'https://service-proxy.bgdi-dev.swisstopo.cloud'
+
+export function appendParamsToUrl(url, paramString) {
+    if (paramString) {
+        const parts = (url + ' ').split(/[?&]/)
+        url +=
+            parts.pop() === ' '
+                ? paramString
+                : parts.length > 0
+                ? '&' + paramString
+                : '?' + paramString
+    }
+    return url
+}
+
+// Test if the URL comes from a friendly site
+export function isAdminValid(url) {
+    return isValidUrl(url) && ADMIN_URL_REGEXP.test(url)
+}
```

Comment:
Same as above, I'm not a big fan of `admin` word in this case, would prefer `bgdi`

Reply:
same